integration/integration_memberlist_single_binary_test.go

```diff
@@ -68,19 +71,19 @@ func startSingleBinary(s *framework.Scenario, name string, join string) error {
 		flags["-memberlist.join"] = join
 	}
 
-	serv := framework.NewService(
+	serv := e2e.NewService(
 		name,
-		framework.GetDefaultCortexImage(),
-		framework.NetworkName,
+		e2ecortex.GetDefaultImage(),
+		e2e.NetworkName,
 		[]int{80, 8000},
 		nil,
-		framework.NewCommandWithoutEntrypoint("cortex", framework.BuildArgs(framework.MergeFlags(ChunksStorage, flags))...),
-		framework.NewReadinessProbe(80, "/ready", 204),
+		e2e.NewCommandWithoutEntrypoint("cortex", e2e.BuildArgs(e2e.MergeFlags(ChunksStorage, flags))...),
```

Comment:
- `e2e.MergeFlags` > `mergeFlags`
- What do you think to map `buildArgs = e2e.BuildArgs` in `integration/util.go` as well?

Reply:
If I can be honest this practice is quite surprising to me. But it's your e2e test, so happy to add those awkward vars. This is for... shorter name?

Reply:
It was for shorter names. With the `e2e` package naming it's less a problem. You can remove the mapping at all.